### ISSUE TITLE
0.2.4 fails to build with GHC 8.6.5 on Windows

### ISSUE THREAD

**Issue by Minoru** [kazu-yamamoto/easy-file#9]
Hello! The latest release fails to build with the following error:

```
 System\EasyFile\Missing.hs:180:10: error:
Error:     Variable not in scope:
      setFilePointerEx :: HANDLE -> Integer -> FilePtrDirection -> IO a0
    |
180 |     _ <- setFilePointerEx hdl (fromIntegral siz) fILE_CURRENT
    |          ^^^^^^^^^^^^^^^^
Error: Process completed with exit code 1.
```

This is because `setFilePointerEx` was added in Win32 2.6.2.0 (https://hackage.haskell.org/package/Win32-2.13.2.1/changelog), and GHC versions up to and including 8.10.5 bundle Win32 2.6.1.0. So the latest release can't be built with anything older than 8.10.6, which is quite unfortunate.

Is there any chance this could be fixed?

**Comment by kazu-yamamoto** [comment]
Sorry for your inconvenience.
I admit that I have poor knowledge for Win32 API.
May I ask you to add `#ifdef` to make `setFileSize` `undefined` for GHC 8.10 or earlier on Windows?

**Comment by kazu-yamamoto** [comment]
GHC 8.10.7 is used on CI.
It's unlucky.
Should I add GHC 8.8 to CI?

**Comment by kazu-yamamoto** [comment]
This catches the reported bug nicely.
https://github.com/kazu-yamamoto/easy-file/tree/ci

**Comment by Minoru** [comment]
> I admit that I have poor knowledge for Win32 API.

Same here. I just have Windows in my CI and noticed the breakage.

> May I ask you to add #ifdef to make setFileSize undefined for GHC 8.10 or earlier on Windows?

I don't mind sending a PR once your ci branch is merged. You can probably do it yourself just as easily: just wrap the function in `#if MIN_VERSION_Win32(2, 6, 2, 0) ... #endif` or something like that.

**Comment by kazu-yamamoto** [comment]
The CI work has been merged.
